docs/kyma/06-01-installation.md

```diff
@@ -77,8 +77,8 @@ The **status.state** field uses one of the following four values to describe the
 |----------|-------------|
 | **Installed** | Installation successful. |
 | **Uninstalled** | Uninstallation successful. |
-| **InProgress** | The Installer is still installing or uninstalling Kyma. No errors logged. |
-| **Error** | The Installer encountered a problem but it continues to try to process the resource. |
+| **InProgress** | The Installer is still installing or uninstalling Kyma. No errors for the current installation step are logged. |
```

Comment:
```suggestion
| **InProgress** | The Installer is still installing or uninstalling Kyma. No errors for the current installation step are logged. |
```

Reply:
IMHO it is important to keep the information, that no errors have been logged for `the current installation step`. What do you think about:
`The process of (un)installing Kyma is running and no errors for the current installation step have been logged.`
?

Reply:
I also wondered about that, but I figured if it was important you'd point that out anyway, and voila :D 
That sounds good, let's go with that, just skip the "installation" in `installation step` :)